2019-12-Montreal/agenda.md

```diff
@@ -16,54 +16,54 @@ Attendees must register [HERE]( https://www.cvent.com/events/node-js-interactive
+- Room 510AC : ~ 120 People
+- Room 513EF : ~ 70 People
+- Room 512DH : ~ 45 People
 
 ## Feedback Forum:
-  TBA
+  Please submit your feedback about what you liked and how can we improve it next time. [Form Link](https://docs.google.com/forms/d/e/1FAIpQLScc_eFLdNswhx-JI9D-O9TyIAqX5GVRaDz9RrSt_QoFlwSbkg/viewform)
 
 ## Collaborator Summit Day One: 13th December, 2019
 
 ### Agenda
-| Time | Main | Issue | Slides |
-|------|------|----------|----------|
-| Zoom | https://zoom.us/j/? | | |
-| 0800 | Welcome and Introduction to Collab Summit | | | |
-| 0830 | Node.js Collection & Social Media | [198](https://github.com/openjs-foundation/summit/issues/198) | | |
-| 0930 | npm open rfc meeting | [197](https://github.com/openjs-foundation/summit/issues/197) | | |
-| 1030 | **BREAK** | | | |
-| 1100 | Node.js Release working group | [195](https://github.com/openjs-foundation/summit/issues/195) | | |
-| 1230 | **LUNCH** | | | |
-| 1330 | An Update on Unflagged ES Modules in Node.js | [207](https://github.com/openjs-foundation/summit/issues/207) | | |
-| 1530 | **BREAK** | | | |
-| 1600 | Node.js Tooling Group Meeting | [194](https://github.com/openjs-foundation/summit/issues/194) | | |
-| 1700 | Package Maintenance working group session | [209](https://github.com/openjs-foundation/summit/issues/209) | | |
-| 1800 | END OF DAY 1 | | | |
+| Time | Room 510AC | Room 513EF | Room 512DH |
+|------|-----------------------------|----------------------------|---------------------------|
+| Zoom | https://zoom.us/j/ | https://zoom.us/j/ | https://zoom.us/j/ |
+| 1000 | Introduction
+| 1030 | Node.js Streams Promise Support [216](https://github.com/openjs-foundation/summit/issues/216) | Node.js Community Committee: OKR building [203](https://github.com/openjs-foundation/summit/issues/203) | Node.js Release working group [195](https://github.com/openjs-foundation/summit/issues/195) |
+| 1130 | npm open rfc meeting [197](https://github.com/openjs-foundation/summit/issues/197) | Node.js Community Committee: OKR building [203](https://github.com/openjs-foundation/summit/issues/203) | |
+| 1230 | 🍍 Lunch
+| 1330 | An Update on Unflagged ES Modules in Node.js [207](https://github.com/openjs-foundation/summit/issues/207) | Node.js Collection & Social Media [198](https://github.com/openjs-foundation/summit/issues/198) | CPC Blue Sky Thinking and Breakout Planning |
+| 1430 | _previous session continues_ | Roadshows and Events for OpenJS Projects [211](https://github.com/openjs-foundation/summit/issues/211) | _previous session continues_
+| 1530 | ☕️ coffee break
+| 1600 | Node.js Tooling Group Meeting [194](https://github.com/openjs-foundation/summit/issues/194) | I'm a lawyer in the Google Open Source Programs office AMA [220](https://github.com/openjs-foundation/summit/issues/220) | AMP Infrastructure Working Group [226](https://github.com/openjs-foundation/summit/issues/226)
```

Comment:
We _really_ need two hours for the tooling group session...

Reply:
or even 90 minutes

Reply:
I think if we tighten the session up a bit, 90 minutes could work great; 60 might be a bit tight, last year we ended up deep ending on a few topics, and ate through 60 minutes quickly.

Reply:
@boneskull Yes, we have space for 2hrs. We can move the tooling session to the second day in the third room. I'll follow up with @evahowe.

Reply:
@WaleedAshraf 😍

Reply:
@boneskull @WaleedAshraf - There isn't a third room on the second day.  There is room in the second room on Saturday afternoon.  Does that work?

Reply:
We are moving your session to day 2 with 90mins.